### DISCUSSION TITLE
Do all pictures need to be reviewed now?

### DISCUSSION THREAD

**Discussion by q849204509** [civitai/civitai#408] [Q&A] [answered]
Do all pictures need to be reviewed now? 
I uploaded a lora，Now all images don't seem to be displaying properly. I didn't upload any nsfw pictures.

**Comment by JustMaier** [comment, marked as answer]
All images are scanned, however it appears that our scanner was having issues over the last 16 hours. We've gotten it fixed up and now things should be appearing as expected.
